Formula/naturaldocs.rb

```diff
@@ -1,21 +1,26 @@
-class Naturaldocs < Formula
+class NaturalDocs < Formula
```

Comment:
You'll need to revert this. `NaturalDocs` here would be a filename of `natural-docs.rb`.